configure

```diff
@@ -435,19 +437,16 @@ def b(value):
 
 def pkg_config(pkg):
   pkg_config = os.environ.get('PKG_CONFIG', 'pkg-config')
-  args = '--silence-errors'
+  args = ['--silence-errors']
   retval = ()
   for flag in ['--libs-only-l', '--cflags-only-I', '--libs-only-L']:
     try:
-      val = subprocess.check_output([pkg_config, args, flag, pkg])
-      # check_output returns bytes
-      val = val.encode().strip().rstrip('\n')
-    except subprocess.CalledProcessError:
-      # most likely missing a .pc-file
-      val = None
-    except OSError:
-      # no pkg-config/pkgconf installed
-      return (None, None, None)
+      proc = subprocess.Popen(shlex.split(pkg_config) + args + [flag, pkg],
```

Comment:
I made it an array so that it's easier to add arguments later if that ever becomes necessary.